Source/WarpX.H

```diff
@@ -552,6 +552,9 @@ protected:
     //! Tagging cells for refinement
     virtual void ErrorEst (int lev, amrex::TagBoxArray& tags, amrex::Real time, int /*ngrow*/) final;
 
+    //! Use this function to override the Level 0 grids made by AMReX
+    virtual void PostProcessBaseGrids (amrex::BoxArray& ba0) const final;
```

Comment:
```suggestion
    //! Use this function to override the Level 0 grids made by AMReX.
    //! This function is called in amrex::AmrCore::InitFromScratch.
    virtual void PostProcessBaseGrids (amrex::BoxArray& ba0) const final;
```

I don't see an explicit call of this function in WarpX. Is this called from `amrex::Am`

AmrMesh::MakeBaseGrids

Reply:
Yes, it is called from `AmrMesh::MakeBaseGrids`, which in turn is called `AmrMesh::MakeNewGrids(Real)`.